build/templates/session.py.mako

```diff
@@ -32,77 +32,70 @@ from ${module_name} import library_singleton
 from ${module_name} import python_types
 
 
-class AttributeViInt32(object):
```

Comment:
Should these move to a separate file instead of being in session.py?

Reply:
I agree. I'll move it on its own PR so as to not make the diff too difficult.